jaas-ljm/example.js

```diff
@@ -164,16 +207,69 @@ function onConnectionSuccess() {
     room.join();
     room.setSenderVideoConstraint(720);  // Send at most 720p
     room.setReceiverVideoConstraint(360);  // Receive at most 360p for each participant
-}
+};
 
 
-function onConnectionFailed() {
-    console.error('Connection Failed!');
-}
+const onConnectionFailed = () => {
+    console.error('connection failed!');
+};
+
+const connect = async () => {
+    const tenant = document.getElementById('tenantInput').value;
+    token = document.getElementById('tokenInput').value;
+    roomName = document.getElementById('roomInput').value;
+    releaseVersion = releaseInput.value;
+
+    options = buildOptions(tenant, roomName, releaseVersion);
+
+    // Initialize lib-jitsi-meet
+    JitsiMeetJS.init(options);
+
+    // Initialize logging.
+    JitsiMeetJS.setLogLevel(options.logging.defaultLogLevel);
+    for (const [loggerId, level] of Object.entries(options.logging)) {
+        if (loggerId !== 'defaultLogLevel') {
+            JitsiMeetJS.setLogLevelById(level, loggerId);
+        }
+    }
+
+    const tracks = await JitsiMeetJS.createLocalTracks({ devices: ['audio', 'video'] });
+    onLocalTracks(tracks);
+
+    connection = new JitsiMeetJS.JitsiConnection(null, token, options);
+    console.log(`using LJM version ${JitsiMeetJS.version}!`);
```

Comment:
@saghul, is this enough of a heads up that the version changed or should I compare it to the previous version?

Reply:
Looks good. Maybe log it at the start, before we call JitsiMeetJS.init?

Reply:
Oh, it will print the same version, because I'm changing the ljm script on release input blur. Should I change that part? I added a reload button next to the input to signal that it's time to reload the connection.

Reply:
Added a log in the blur handler to notify the upcoming version change.